gha/bin/release-init.py

```diff
@@ -221,6 +227,36 @@ def handle(self, base: str, head: str) -> NoReturn:
                     inwardIssue=issue_key,
                     outwardIssue=release_key,
                 )
+        # Check for previous cancelled release
+        cancelled_release_scope = []
+        if last_cancelled_release:
+            # Get diff between current and latest Cancelled release
+            cancelled_sha = cancelled_tag.commit.sha
+            commits = self._get_commits_from_diff(cancelled_sha, sha)
+            # Read commit messages aka task titles
+            for commit in commits:
+                title = commit.commit.message.split('\n')[0]
+                if title == "Merge branch 'main' into release":
+                    continue
+                cancelled_release_scope.append(title)
+            # Logout ticket titles from latest canceled release
+            self.log.info(f'Latest (cancelled) release scope contains {len(cancelled_release_scope)} tasks:')
```

Comment:
Да кто сказал, что last всегда будет cancelled? Или cancelled или published - сегодня утром я так говорил.

```suggestion
            self.log.info(f'Latest release scope contains {len(cancelled_release_scope)} tasks:')
```